data/human/news.txt

```diff
@@ -2150,3 +2150,346 @@ news "graffiti on spaceport wall"
+			"supporting local protests"
+		word
+			". "
+		word
+			"When "
+		word
+			"the planetary authorities "
+			"a gang of local pirates "
+		word
+			"accost "
+			"confront "
+		word
+			"them and demand that they "
+		word
+			"disperse"
+			"make themselves scarce"
+			"abandon their objective"
+			"shut up and buzz off"
+		word
+			", "
+		word
+			"the guerrillas "
+			"the group "
+		word
+			"bluster at them "
+			"saber-rattle "
+		word
+			"in response"
+		word
+			", but they end up dispersing."
+			", and the two groups back down."
+
+phrase "Pronoun"
+	word
+		"his"
+		"her"
+		"their"
+
+phrase "Careless Ends Well"
+	word
+		"turns around to reenter the ship"
+	word
+		" as it's struck by a wave, nudging it close to the edge of the platform. The captain then moves the ship to a lowerable pad."
+		", moving it to a lowerable pad before it could be caught by a wave."
+
+phrase "Careless Ends Poorly"
+	word
+		"ignores air traffic control and heads to the bar. The ship"
+		"argues with the controller about their landing. The ship"
+		"turns around to reenter the ship. However, it"
+	word
+		" is "
+	word
+		"struck "
+		"caught "
+	word
+		"by a wave and "
+	word
+		"swept into the sea."
+		"pushed off the pad."
+
+news "Careless Landing"
+	location
+		planet "Jakobsen"
+	name
+		word
+			"Recklessness"
+	message
+		word
+			"A Sparrow"
+			"An Enforcer"
+			"A Shuttle"
+			"A Heavy Shuttle"
+			"A Star Barge"
+			"A Berserker"
+			"A Fury"
+			"A Hawk"
+			"A Wasp"
+		word
+			" lands on a pad meant for heavier ships. Air traffic control is immediately upon the captain to move ${Pronoun} ship. The captain "
+		phrase
+			"Careless Ends Well"
+			"Careless Ends Poorly"
+
+phrase "CCOR Song Description"
+	word
+		"a song about "
+	word
+		"community and everyone pitching in where they can"
+		"the tyranny of a man name Bailey, until he was overthrown"
+		"the CCOR, praising their accomplishments"
+		"resistance and opposition to the current planetary administration"
+
+phrase "Worker's Demands"
+	word
+		"safety regulations"
+		"increased pay"
+		"restoration of the CCOR"
+		"justice against planetary security brutality"
+		"immediate repositioning of the spaceport"
+
+news "Discontent"
+	location
+		planet "Bailey"
+	name
+		word
+			"Conflict"
+	message
+		word
+			"Allegations that planetary authorities will no longer move the spaceport has provoked "
+		word
+			"large scale "
+			"mass "
+			"moderate "
+			"scattered "
+		word
+			"riots. "
+			"protests. %"
+		word
+			"The rioters are "
+		word
+			"singing ${CCOR Song Description}. "
+			"chanting a list of their demands, including ${Worker's Demands}. "
+			"sizing up the planetary security forces. "
+		word
+			"Security eventually steps in, "
+		word
+			"but they're pushed back. "
+			"but they're routed. "
+			"and they disperse the demonstrators. "
+			"and the demonstrators scatter but don't appear dissuaded."
+		replace
+			"% The rioters" "The protesters"
```

Comment:
```suggestion
			"%The rioters" "The protesters"
```

Reply:
I agree. Was thinking about that for a while. Been meaning to make those edits but haven't had motivation for the past little while.